package.json

```diff
@@ -1,19 +1,30 @@
 {
-    "name": "infusion-docs",
-    "version": "1.5.0",
-    "description": "Infusion Documentation",
-    "license" : "CC-BY-3.0",
-    "repository": "https://github.com/fluid-project/infusion-docs",
-    "scripts": {
-        "docpad": "docpad run --env static",
-        "deploy": "docpad deploy-ghpages --env static"
-    },
-    "dependencies": {
-        "URIjs": "^1.14.1",
-        "docpad-plugin-ghpages": "~2.4.4",
-        "docpad-plugin-handlebars": "~2.3.0",
-        "docpad-plugin-highlightjs": "~2.2.2",
-        "docpad-plugin-marked": "~2.2.1",
-        "fs-extra": "^0.18.0"
-    }
+  "name": "infusion-docs",
```

Comment:
Is there a way you could get your editor to revert to 4-spacing in JSON files? Unfortunately our linter isn't good enough to pick this up, but it is what we use in other files and it tends to make unnecessary diffs when, e.g. package.json files get edited

Reply:
Yes, there is, and I updated the setting.  There may be a few more unnecessary diffs as various pieces of work cross to the other side.